pyevtk/hl.py

```diff
@@ -100,11 +101,17 @@ def imageToVTK(path, origin = (0.0,0.0,0.0), spacing = (1.0,1.0,1.0), cellData =
     if cellData != None:
         keys = list(cellData.keys())
         data = cellData[keys[0]]
-        end = data.shape
+        if hasattr(data,'shape'):
+            end = data.shape
+        elif(data[0].ndim==3 and data[1].ndim==3 and data[2].ndim==3):
```

Comment:
been forever since I've looked at this code but doesn't this assume 3D vector arrays?

Reply:
I guess it does - I only thought about using it for 3D vector fields like velocity etc.

Would it be worth generalising it to higher dimensional vectors?

Reply:
I think it should support whatever the format supports but I've long passed the reigns to @renefritze so I'll let him decide